packages/flame_test_helpers/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Dont we use symlinks to reference the LICENSE now?

Reply:
Right, forgot that we haven't changed away from that yet (which we should, since it doesn't work to publish on linux when it is a symlink)

Reply:
Oh, seems like we did move away from it?